webapp/Component.js

```diff
@@ -1,6 +1,6 @@
-sap.ui.define(["sap/ui/core/UIComponent"], function(UIComponent){
+sap.ui.define(["sap/ui/core/UIComponent", "sap/ui/core/ComponentSupport"], function(UIComponent) {
```

Comment:
shouldn't it be identified by the bootstrap tag automatically? i'm preety sure app developers will forget to load this dependency frequently if it has to be specified again here in the Component

Reply:
it should, but it isn't. I think Matthias added the dependency as a workaround.

Reply:
Exactly. See https://github.com/SAP/ui5-builder/issues/92

Reply:
thanks for clarifying, fine for me